pkg/operation/cloudbotanist/localbotanist/addons.go

```diff
@@ -52,5 +52,11 @@ func (b *LocalBotanist) GenerateAdmissionControlConfig() (map[string]interface{}
 
 // GenerateNginxIngressConfig generates values which are required to render the chart nginx-ingress properly.
 func (b *LocalBotanist) GenerateNginxIngressConfig() (map[string]interface{}, error) {
-	return common.GenerateAddonConfig(nil, b.Shoot.Info.Spec.Addons.NginxIngress.Enabled), nil
```

Comment:
As transporting the `loadBalancerSourceRanges` to the Helm chart seems to be not cloud specific, please move this part to https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L147 and keep the cloud specific implementations untouched.